Yoast/YoastSEO.js
# Add YOAST_RECALIBRATION environment variable to the example

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Added `YOAST_RECALIBRATION` environment variable that defaults to `disabled` to the webpack example.
* Added package.json script `start-recalibration` that does the same as start, only with first setting `YOAST_RECALIBRATION` to `enabled`.
* Note you can also change it with a `.env.development` file that includes that variable: `YOAST_RECALIBRATION=enabled`. For more info see: https://github.com/motdotla/dotenv#usage

## Test instructions

This PR can be tested by following these steps:

Issue #1924 uses the environment variable. Therefore, this is only really visible after that PR is merged. In the meantime, you can add a `console.log` in, for example, an assessment, to see if it works. Something like this:
```
if ( process.env.YOAST_RECALIBRATION === "enabled" ) {
	console.log( "Assessment uses recalibration!" );
} else {
	console.log( "Assessment does not use recalibration!" );
}
```

See #1913 for an overview of what will be in recalibration.

* Go to the webpack example `examples/webpack` and run `yarn`.
* Start the example without recalibration: `yarn start`.
* Ensure the analysis is *without* recalibration.
* Start the example with recalibration: `yarn start-recalibration`.
* Ensure the analysis is *with* recalibration.

Fixes #1931
